python-checks/src/test/resources/checks/regex/groupReplacementCheck.py

```diff
@@ -0,0 +1,36 @@
+import re
```

Comment:
I think there is an empty python file that has been commited by mistake (i.e. python-checks/src/test/resources/checks/regex/groupReplacementEmptyCheck.py)